maintainers/maintainer-list.nix

```diff
@@ -16487,6 +16487,13 @@
     githubId = 28323;
     name = "Peter Simons";
   };
+  petrkozorezov = {
+    email = "petr.kozorezov@gmail.com";
+    github = "petrkozorezov";
+    githubId = 645017;
+    name = "Petr Kozorezov";
+    keys = [ { fingerprint = "7F1A 353D 3D6D 9CEF 63A9  B5C6 699F 32D5 9999 7C90"; } ];
```

Comment:
I can't seem to receive the key properly, `new key but contains no user ID - skipped`.

Reply:
It was an issue with key email verification.
Should work now.

Reply:
Works fine now, thanks.